src/core_plugins/metric_vis/public/__tests__/metric_vis.js

```diff
@@ -0,0 +1,60 @@
+import $ from 'jquery';
+import ngMock from 'ng_mock';
+import expect from 'expect.js';
+
+import VisProvider from 'ui/vis';
```

Comment:
likely named import here 

`import {VisProvider}`

is reason of failing test